docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md

```diff
@@ -176,67 +191,62 @@ After a failover, do the following:
 
 ## Upgrade instances
 
-All instances involved in HA should use the same version of YugabyteDB Anywhere. Promoting a standby instance using a YBA backup from an active instance where the YBA versions are different may result in errors.
+All instances involved in HA should use the same version of YugabyteDB Anywhere. This ensures that, in steady state operation, all instances run the same version of YugabyteDB Anywhere. You will receive alerts if a mismatch is detected between active and standby instances.
 
-Although you can upgrade all YBA instances simultaneously and there are no explicit ordering requirements regarding upgrades of active and standby instances, you should follow these general guidelines:
+When upgrading YBA in a HA cluster, upgrade the standby instances first. If the active is upgraded first, state backups will stop replicating to any lower version standbys, causing HA to stop.
 
-- Start an upgrade with the active instance.
-- After the active instance has been upgraded, ensure that YBA is reachable by signing in and checking various pages.
-- Proceed with upgrading standby instances.
-- After upgrading all instances, verify that the standbys are receiving new backups from the active instance.
+Upgrade instances in an HA configuration as follows:
 
-Certificates in the trust store should not require setup again.
+1. Upgrade the standby instances.
 
-{{< warning title="Important: Don't promote an old active backup" >}}
+1. After upgrading a standby instance, ensure that YBA is reachable by signing in and checking various pages.
 
-Immediately after an upgrade, older backups of the active instance _before it was upgraded_ will still be available on the standby. These are not deleted until the standby is promoted at some point, or they expire. Because these old backups are present, you need to be cautious promoting the standby in the time immediately after upgrading the HA pair.
+1. Validate that replication is successful and the standby is receiving backups from the active.
 
-Only promote a standby when both standby and active are at same version, and use the most recent backup that you are confident was received after the active was upgraded.
+1. After all standbys have been upgraded, proceed with upgrading the active instance.
 
-{{< /warning >}}
+After upgrading all instances, verify that the standbys are still receiving new backups from the active instance.
 
-### Upgrade instances in HA
+Certificates in the trust store should not require setup again.
 
-To upgrade instances in a HA cluster, do the following:
+If you are promoting a YBA standby that is running version 2024.1.0 or later, while the old active instance is running a version earlier than 2024.1.0, see the [Limitations](#limitations).
```

Comment:
Let's highlight that this should only be done in extreme scenarios and refer them back to where we talk about version mismatches and how active/standby should be on the same version and upgraded together. They will need to reach out to support

Reply:
Oh I see you called that out in the Limitations, nevermind.